easybuild/easyconfigs/l/libunistring/libunistring-0.9.7-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
```

Comment:
I meant that #4930 has been merged already, so this shouldn't be required anymore?